src/site/stages/build/index.js

```diff
@@ -49,7 +49,7 @@ function defaultBuild(BUILD_OPTIONS) {
   smith.metadata({
     buildtype: BUILD_OPTIONS.buildtype,
     hostUrl: BUILD_OPTIONS.hostUrl,
-    enabledFeatureFlags: BUILD_OPTIONS.enabledFeatureFlags,
+    enabledFeatureFlags: BUILD_OPTIONS.cmsFeatureFlags,
```

Comment:
I decided to leave `enabledFeatureFlags` as a variable available in the liquid templates. I don't know if that's a good call or not yet, to be honest. I'm not a fan of the difference in name (`cmsFeatureFlags` -> `enabledFeatureFlags`), but I didn't think the distinction was necessary in the liquid templates.

I'm open to changing this as well and updating the templates as (if? :thinking:) needed.

Reply:
Shoulder Shrug emoji.